administrator/components/com_content/models/article.php

```diff
@@ -680,6 +680,39 @@ public function featured($pks, $value = 0)
 		return true;
 	}
 
+	public function shareToken($token)
+	{
+
+		// perform whatever you want on each item checked in the list
+		$jinput = JFactory::getApplication()->input;
+
+		$jform = $jinput->getArray(array('jform' => array('title' => 'string')));
+		$title = $jform['jform']['title'];
+
+		// Get a db connection.
+		$db = $this->getDbo();
+
+		// Create a new query object.
+		$query = $db->getQuery(true);
+
+		// Insert columns.
+		$columns = array('articleId', 'title', 'sharetoken');
+
+		// Insert values.
+		$values = array($db->quote($jinput->get("id")), $db->quote($title), $db->quote($token));
```

Comment:
The ID should be passed as parameter in the function and retrieved in the controller. Also use single quotes instead of double quotes.